### PR TITLE
Add an env.sample

### PR DESCRIPTION
**Pull request by jasonLaster**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,34 +12,37 @@ locally. The process is really simple and we accept anyone who's interested in
...
+   $ npm install
+   ```
 2. Copy your Airtable API key from this page: [Airtable account][airtable-account]
 3. Create a `.env.development` file at the root of the project
    replacing `YOUR_KEY_HERE` by the key you copied in the previous step:
-    ```bash
-    $ echo "AIRTABLE_API_KEY=YOUR_KEY_HERE" > .env.development
-    ```
+   ```bash
+   $ cp .env.sample .env.development
```
Comment (jasonLaster) [inline README.md]: not sure if this was clear enough

File: .env.sample [hunk 2]
```diff
@@ -0,0 +1,3 @@
+AIRTABLE_API_KEY=YOUR_KEY_HERE
+GATSBY_ALGOLIA_APP_ID=9W9HIJ1XKH
+GATSBY_ALGOLIA_SEARCH_KEY=f31eb728872ce9b2be62efd46c3486fb
```
Comment (jasonLaster) [inline .env.sample]: So, these are my keys... @bloudermilk what are your keys?

Comment (bloudermilk) [reply .env.sample]: @jasonLaster here are the climatescape public keys:
```
GATSBY_ALGOLIA_APP_ID=SKYZAC71CL
GATSBY_ALGOLIA_SEARCH_KEY=5d4b018d3b45f2db1398c038c46cfa8a
```

File: README.md (hunk 1 above)
Comment (bloudermilk) [reply README.md]: Seems good to me 👍

**Pull request merged**